cmd/lifecycle/analyzer.go

```diff
@@ -130,6 +136,17 @@ func (a *analyzeCmd) Privileges() error {
 	return nil
 }
 
+func (aa *analyzeArgs) registryImages() []string {
```

Comment:
Needed to change the receiver to `analyzeArgs` (from `analyzeCmd`) so I moved it up in the file.